util/ranger/ranger_test.go

```diff
@@ -1018,6 +1018,15 @@ func (s *testRangerSuite) TestColumnRange(c *C) {
 			resultStr:   "[[\"a\",\"a\"]]",
 			length:      1,
 		},
+		// This test case cannot be simplified to [1, 3] otherwise the index join will executes wrongly.
```

Comment:
Use this test to assert that there won't be range merging in this case.

Reply:
Why can't merge to [1,3]? Because of the prefix index?